src/components/cards/DataSummaryCardGroup.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import CountBox from './CountBox';
+
+const DataSummaryCardGroupWrapper = styled.div`
+  display: flex;
+  &.connected-count-card-wrapper {
+    width: 100%;
+  }
+`;
+
+const SummaryCard = styled.div`
+  display: inline-block;
+  background-color: #ffffff;
+  height: 97px;
+  flex-grow: 1;
+  flex-basis: ${props => props.widthPercentage}%;
+  .separated-data-summary-card-group & {
+    margin-left: 15px;
+    &:first-child {
+      margin-left: 0px;
+    }
+  }
+  .connected-data-summary-card-group & {
+    margin: 0;
+    &:first-child {
+      border: none;
+    }
+  }
+  position: relative;
+`;
+
+const SummaryCardLeftBorder = styled.div`
+  position: absolute;
+  top: 7px;
+  bottom: 7px;
+`;
+
+const SubCountBoxesWrapper = styled.div`
+  display: flex;
+  height: 100%;
+`;
+
+const SubCountBoxWrapper = styled.div`
+  flex-grow: 1;
+  height: 100%;
+`;
+
+const SummarySubCardLeftBorder = styled.div`
+  position: absolute;
+  top: 28px;
+  bottom: 28px;
+`;
+
+/**
+ * Little card with a bunch of counters on it for cases, experiments, files, ...
+ */
+class DataSummaryCardGroup extends Component {
+  render() {
+    const widthPercentage = 100 / this.props.summaryItems.length;
+    return (
+      <DataSummaryCardGroupWrapper className={'data-summary-card-group '.concat(this.props.connected ? 'connected-data-summary-card-group' : 'separated-data-summary-card-group')}>
```

Comment:
I think we should write a test for this
Also I don't think the `'data-summary-card-group'` class name is necessary, just the connected vs. not connected.